src/core/Hotkey.ts

```diff
@@ -40,11 +40,15 @@ const _hotkeys_desc: { [key: string]: string } = {};
 const _namespaces: {[key: string]: HotkeyNamespace} = {};
 const _destructors: (() => void)[] = [];
 
-keymaster.filter = function(event) {
+const translateNumpad = (event:any) => 
+  document.dispatchEvent(new KeyboardEvent('keydown', { keyCode:  event.keyCode - 48 }));
```

Comment:
Can we use keycode constants instead of just the numbers so it is easier to see these ranges and what we are translating?